packages/contracts/contracts/ionic/levered/LeveredPosition.sol

```diff
@@ -163,6 +165,13 @@ contract LeveredPosition is LeveredPositionStorage, IFlashLoanReceiver {
     }
   }
 
+  function rescueTokens(IERC20Upgradeable asset) external {
+    if (msg.sender != factory.owner()) revert OnlyFactoryOwner();
+    if (asset == stableAsset || asset == collateralAsset) revert AssetNotRescuable();
+
+    asset.transfer(positionOwner, asset.balanceOf(address(this)));
```

Comment:
Should we make sure position is closed first?

Reply:
I wouldn't worry about that, since it's an admin only fn

Reply:
OK sounds good!